### ISSUE TITLE
Error popup when deleting module instance on Moodle 3.1 / Totara 12

### ISSUE THREAD

**Issue by logan-reynolds** [blindsidenetworks/moodle-mod_bigbluebuttonbn#126]
The following error is encountered in a popup modal when deleting a module instance within a course on both Moodle 3.1 and on Totara 12 (which is based on Moodle 3.3):
```
error/cannotdeletemoduleinstance
URL: <OMITTED>
Debug info: Cannot delete the module bigbluebuttonbn (instance). Error code: cannotdeletemoduleinstance $a contents:
Stack trace:

* line 1202 of /course/lib.php: moodle_exception thrown
* line 171 of /course/rest.php: call to course_delete_module()
```
Note the module instance *does* get deleted and nothing else seems wrong so this is probably low priority, but I'm submitting this issue to determine why this error is occurring on lower Moodle/Totara versions but does *not* occur on more recent Moodle 3.5.

**Comment by jfederico** [comment]
This bug seems to be this one https://tracker.moodle.org/browse/CONTRIB-7660 it was already addressed with PRs #136 and #137 depending on the version you are using.
Can you give it a try with the latest code and confirm?

**Comment by jfederico** [comment]
I just got confirmation from the guy who open that ticket that the latest code fixed the issue. You can take the new code from master or v2.21-stable depending on what version you are. I am going to close it as we are just about to release these versions this week.

Thanks for the report
